cmd/ignite/cmd/root.go

```diff
@@ -117,6 +117,14 @@ func NewIgniteCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	return root
 }
 
+func isNonRootCommand(cmd string, parentCmd string) bool {
+	isVersionCmd := cmd == "version" && parentCmd == "ignite"
+	isHelpCmd := cmd == "help" && parentCmd == "ignite"
+	isImageCmd := cmd == "image" && parentCmd == "ignite"
+	isKernelCmd := cmd == "kernel" && parentCmd == "ignite"
+	return isVersionCmd || isHelpCmd || isImageCmd || isKernelCmd
```

Comment:
This could be simplified using a switch statement. Something like:

```go
if parentCmd != "ignite" {
        return false
}

switch cmd {
    case "version", "help", "image", "kernel":
        return true
}

return false
```

Reply:
Done